packages/@expo/metro-config/src/serializer/serializeChunks.ts

```diff
@@ -189,19 +190,22 @@ class Chunk {
         });
   }
 
+  private getStableChunkSource(serializerConfig: Partial<SerializerConfigT>) {
+    return this.options.dev
+      ? ''
+      : this.serializeToCodeWithTemplates(serializerConfig, {
+          // Disable source maps when creating a sha to reduce the number of possible changes that could
+          // influence the cache hit.
```

Comment:
Is this safe for the Debug IDs.  I'm thinking if the source map changes the debug id should too. But for the source map to change the source would have to first. Is that the right assumption?

Reply:
This is effectively concatenating the bundle but with source map annotations and async chunks removed. We use this for bundle splitting to replicate the standard bundler behavior, where async references live in a different file.